editing/edit-features-with-feature-linked-annotation/src/main/java/com.esri.samples.edit_features_with_feature_linked_annotation/EditAttributesDialog.java

```diff
@@ -67,6 +67,7 @@
         } catch (Exception e) {
           e.printStackTrace();
         }
+        return true;
       }
       return null;
```

Comment:
Perhaps it's possible to return `false` here. That way we can convert the type `Optional<Boolean>` to just `Boolean`.

Reply:
`Dialog.showAndWait()` returns an Optional, so I don't think I can use a `Boolean`.

Reply:
Ah no worries then!